frontend/src/GameModule/Gacha/components/GameView.vue

```diff
@@ -62,10 +65,15 @@ const onRoll = () => {
     return;
   }
 
-  axios
-    .get(`/game/${selectedGame.value.id}/gacha/?format=json&numrolls=${numRolls.value}`)
+  callApi<Array<Roll>>({
```

Comment:
Can you update places where it is calling callApi to be: const response = await callApi... instead of doing .then?